package.json

```diff
@@ -22,16 +22,20 @@
   "homepage": "https://github.com/blackbaud/skyux-builder-config#readme",
   "dependencies": {
     "browserstack-local": "1.3.0",
+    "hub-utility": "AnkurGel/selenium-hub-utilities",
```

Comment:
I'd rather not point to someone's GitHub page.  If we're only using the `keep-alive` (fast-selenium) script, I think I'd rather just include that in this library.  https://github.com/browserstack/fast-selenium-scripts/blob/master/node/fast-selenium.js